templates/admin-page.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Template for the Admin page.
+ *
+ * @package plugin-check
+ */
+
+if ( empty( $available_plugins ) ) {
+	return;
+}
+?>
+
+<div class="wrap">
+
+	<h1><?php esc_html_e( 'Plugin Check', 'plugin-check' ); ?></h1>
+
+	<div class="card">
```

Comment:
Why display this content in a card?

Reply:
I thought it will look good in card (like below screenshot), but I have removed it now we can circle back on this when we actually implement output of the checks.

![Screenshot 2023-02-20 at 12 39 35 PM](https://user-images.githubusercontent.com/9035925/220104934-30732e20-d906-47db-b795-22dc1353cd84.png)